autoprofile.py

```diff
@@ -17,6 +17,8 @@
 import asyncio
 import time
 import logging
+from PIL import Image
```

Comment:
Put that in a try..catch

Reply:
It just errors once on starting in console, same with kanging. If I execute `.autopfp` it does nothing. Do I really need to catch?

Reply:
Do it.